src/entity.js

```diff
@@ -570,6 +574,17 @@ Entity.configure = function(options) {
  *   },
  * }
  *
+ * To use an in-memory, testing-oriented table, replace the `account` key with
+ * `inMemory`, and do not supply credentials:
+ * {
+ *   inMemory: true,
```

Comment:
Suggestion instead of specifying `inMemory: true` we could use `account: 'in-memory'` so it becomes of a special case of specifying `account`, `table` and taskcluster `credentials`.

This way our config files become a lot simpler... As we just specify `account: 'in-memory'` and in production `account: 'taskclusterqueue'`...